src/cmdline.cpp

```diff
@@ -228,24 +228,24 @@ namespace boost { namespace program_options { namespace detail {
 
         if (m_additional_parser)
             style_parsers.push_back(
-                boost::bind(&cmdline::handle_additional_parser, this, _1));
+                boost::bind(&cmdline::handle_additional_parser, this, boost::placeholders::_1));
```

Comment:
Why not `using namespace boost::placeholders;` here as well?

Reply:
There's just one placeholder, so qualifying the namespace doesn't take much space. I can change it to a `using` directive, if you like.

Reply:
Actually, those qualifications are no longer needed. I've removed them.

Reply:
I don't understand; why are they not needed?

Reply:
There was https://github.com/boostorg/program_options/pull/92 that was merged. It added `using namespace boost::placeholders;` directives at the top of the files, which made explicit namespace qualifications in this PR no longer needed.

This PR is still useful as it removes `#include <boost/bind.hpp>` in one place and removes the usage of `std::bind1st` in another.